client/pages/_app.tsx

```diff
@@ -5,11 +5,15 @@ import { PaginatorContextProvider } from "../context/PaginatorContext";
 import { FetchContextProvider } from "../context/FetchContext";
 
 import "../styles/normalize.css";
-import "../styles/application.css";
-import "../styles/client.css";
 import "../styles/justgrid.css";
+import "../styles/globals.css";
 import "@fortawesome/fontawesome-free/css/all.min.css";
+
+import "../styles/application.css";
+import "../styles/client.css";
+
 import LogoutButton from "../components/LogoutButton";
+import ThemeMenu from "../components/ThemeMenu";
```

Comment:
You import the ThemeMenu, but you don't use it. Indeed, I don't see a theme menu anywhere in the application at this point, though I do notice that the "log out" button has moved from right-aligned to left-aligned for some reason.